### PR TITLE
Revert lint rule changes

### LINKED ISSUE #17425: AndroidManifest.xml: handle `minSdk 24` changes for  `dropanddrop`
I've reverted the `AndroidManifest.xml` `dropanddrop` changes.

The upgrade should be extracted to another issue

_Originally posted by @david-allison in https://github.com/ankidroid/Anki-Android/issues/17364#issuecomment-2468146738_

### PR DESCRIPTION
**Pull request by jradchaima**
## Purpose / Description
*   #17425
The lint build fails with the following error: "Error: For locale "ca" (Catalan) the following quantity should also be defined: many "[MissingQuantity] to solve this error I Revert lint rule changes [UnusedQuantity, MissingQuantity] from fatal to ignore 
## Fixes
*  Fixes #17425 

## Approach
I made modifications to two AndroidManifest.xml
*AnkiDroid/src/main/AndroidManifest.xml
*testlib/src/main/AndroidManifest.xml
in lint-release.xml I changed thes two line from
```xml
<issue id="UnusedQuantity" severity="fatal" />
<issue id="MissingQuantity" severity="fatal" />
```
to
```xml
<issue id="UnusedQuantity" severity="ignore" />
<issue id="MissingQuantity" severity="ignore" />

```
These changes fix the lint build failure, making the build successful
## How Has This Been Tested?

tested on emulator Google HPE device

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by mikehardy — CHANGES_REQUESTED** [review]
oh wait - this is incomplete, it has two parts and the other is to implement the suggestion in this as well: https://github.com/ankidroid/Anki-Android/commit/292d662d0b676edf21939b15c88dcf38fb3d83ba

that is, to do uses-feature there, and ensure the build still passes

**Comment by jradchaima** [comment]
@mikehardy I updated two AndroidManifest.xml files, and the lint build was successful when tested in Android Studio. However, the commit with the fix for the AndroidManifest.xml had some checks that were unsuccessful. I’m not sure what caused the issue, so I’m currently looking on it

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,14 +22,12 @@
...
 
-<!--    At the time of writing, ankidroid minSdk is 23. This library requires API 24.-->
-<!--    In order to use it, we use <uses-sdk tools:overrideLibrary instead of-->
-<!--    <uses-feature android:name. We must ensure this is never called when running on API 23.-->
-    <uses-sdk tools:overrideLibrary="androidx.draganddrop"/>
 
     <uses-feature android:name="android.hardware.audio.output" android:required="false" />
     <uses-feature android:name="android.software.app_widgets" android:required="false" />
     <uses-feature android:name="android.hardware.microphone" android:required="false" />
+    <!--    <uses-feature android:name. We must ensure this is never called when running on API 23.-->
```
Comment (mikehardy) [inline AnkiDroid/src/main/AndroidManifest.xml:29]: this comment is no longer valid - can you remove it?

**Review by mikehardy — CHANGES_REQUESTED** [review]
looks great - just one remaining item about a comment that no longer applies

File: AnkiDroid/src/main/AndroidManifest.xml (hunk 1 above)
Comment (jradchaima) [reply AnkiDroid/src/main/AndroidManifest.xml:29]: yes of course

**Comment by jradchaima** [comment]
I made some errors while updating the branch with the latest changes. I've resolved them and ensured everything is now up-to-date. Please let me know if there's anything else you'd like me to check!

**Comment by mikehardy** [comment]
The "proper" way to update the branch in AnkiDroid is never to do a merge commit - for future reference. We do a rebase-merge strategy here which is a bit unusual based on my experience interacting with other developers but results in a very very clean commit history, which is something we value

Happy to accept the merge commit this time and do a squash-merge of this PR - no problems - but if you plan to contribute to AnkiDroid in the future maybe do some test PR work on your personal Anki-Android repo fork where you locally rebase a PR pulling changes into your branch that way, then force-push back out to your branch. A bit difficult to get used to at first but then pretty easy after some practice

**Review by mikehardy — APPROVED** [review]
looks exactly correct now, thank you!

**Comment by jradchaima** [comment]
Thank you so much for the advice!

**Pull request merged**